package.json

```diff
@@ -7,7 +7,7 @@
     "================================================================================"
```

Comment:
Make sure to update the dependency of auro-icon to use the new released version, rather than the beta version